africanlii/views.py

```diff
@@ -16,6 +17,21 @@ def get_context_data(self, **kwargs):
         )
 
 
+class AGPLegalInstrumentListView(FilteredDocumentListView):
+    model = CoreDocument
+    template_name = "peachjam/legal_instrument_list.html"
+
+    def get_queryset(self):
```

Comment:
I think this must be get_base_queryset()

Reply:
Changed. Is get_base_queryset a django builtin method or is it our own?

Reply:
Our own, in FilteredDocumentListView. The base queryset is what we do faceting and counts from, before filters are applied. `get_queryset()` is the filtered base queryset.